### PR TITLE
Created ashishtom.json

### PR DESCRIPTION
**Pull request by alceil**
Fixes #

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/207"></a>

### CODE REVIEW

**Review by MukulKolpe — CHANGES_REQUESTED** [review]
Hey @alceil, Please don't commit  package-lock.json and package.json.

**Comment by alceil** [comment]
Oh😅 so shall i exclude it and commit again?

**Comment by alceil** [comment]
I have excluded it and I have done the necessary changes. Can you have a look?

File: .gitignore [hunk 1]
```diff
@@ -21,6 +21,9 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
+package.json
```
Comment (schmelto) [inline .gitignore:25]: Please do not exclute them in the gitignore :)

**Review by schmelto — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (alceil) [reply .gitignore:25]: Ok I will make the changes again then😅

Comment (alceil) [reply .gitignore]: I have done all requested now can you have a  look🙂

File: .gitignore [hunk 2]
```diff
@@ -22,5 +22,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+
+
```
Comment (schmelto) [inline .gitignore:26]:
```suggestion
```

**Comment by schmelto** [comment]
Please further exclude package-json and package-json.lock from the pr (only exclute them in the pr not in the gitignore :) )

**Comment by alceil** [comment]
Ok got it 🙂@schmelto

**Comment by alceil** [comment]
Now I have done everything that needed to be merged right?. Or is there anything else? @schmelto

**Comment by schmelto** [comment]
> Now I have done everything that needed to be merged right?. Or is there anything else? @schmelto

No please delete those three files from the pr 
image

only your username.json should be in the pr :)

**Comment by alceil** [comment]
okay got it

**Comment by alceil** [comment]
Done. I just added JSON file in the pr just as you told me to do. @schmelto

**Comment by schmelto** [comment]
Unfortunately the other changes are still in this pr.

you can undo the other files as follows:

> The instructions below assume that the unwanted updates are in a file that already exists in the GitHub repository. If the unwanted updates are in an entirely new file that you created, then you can delete the file from your file system, then run git rm <file path> followed by git commit to remove the file from the PR.
[here](https://ffeathers.wordpress.com/2020/03/21/removing-an-updated-file-from-a-pr-on-github/)

**Comment by alceil** [comment]
Okay got it

**Pull request closed without merging**